eclipse-collections/src/main/java/org/eclipse/collections/impl/map/mutable/ConcurrentMutableHashMap.java

```diff
@@ -164,9 +166,9 @@ public V remove(Object key)
     }
 
     @Override
-    public Set<K> keySet()
+    public MutableSet<K> keySet()
     {
-        return this.delegate.keySet();
+        return SetAdapter.adapt(this.delegate.keySet());
```

Comment:
@johnjstuart you can use `Sets.adapt()` here.

Reply:
Done 👍